cmd/install.go

```diff
@@ -838,8 +838,27 @@ func installDupeDetection(ctx context.Context, config *configs.Config) (err erro
 		return err
 	}
 
-	subCmd := []string{"-m", "pip", "install"}
-	subCmd = append(subCmd, constants.DependenciesDupeDetectionPackages...)
+	// Download dependency list from download page
+	log.WithContext(ctx).Info("Downloading dd-server dependency list...")
+
```

Comment:
No, requirements file (in, txt or whatever) is included into dd-server zip package, no other downloads are needed

Reply:
Noted. Let me revise with the new commit.